code/game/objects/items/stacks/wrap.dm

```diff
@@ -110,48 +110,68 @@
 		return
 
 	if(isitem(target))
-		var/obj/item/I = target
-		if(!I.can_be_package_wrapped())
+		var/obj/item/item = target
+		if(!item.can_be_package_wrapped())
+			balloon_alert(user, "The [target] can not be wrapped!")
 			return
-		if(user.is_holding(I))
-			if(!user.dropItemToGround(I))
+		if(user.is_holding(item))
+			if(!user.dropItemToGround(item))
 				return
-		else if(!isturf(I.loc))
+		else if(!isturf(item.loc))
 			return
 		if(use(1))
-			var/obj/item/delivery/small/P = new(get_turf(I.loc))
-			if(user.Adjacent(I))
-				P.add_fingerprint(user)
-				I.add_fingerprint(user)
-				user.put_in_hands(P)
-			I.forceMove(P)
-			var/size = round(I.w_class)
-			P.name = "[weight_class_to_text(size)] parcel"
-			P.w_class = size
+			var/obj/item/delivery/small/parcel = new(get_turf(item.loc))
+			if(user.Adjacent(item))
+				parcel.add_fingerprint(user)
+				item.add_fingerprint(user)
+				user.put_in_hands(parcel)
+			item.forceMove(parcel)
+			var/size = round(item.w_class)
+			parcel.name = "[weight_class_to_text(size)] parcel"
+			parcel.w_class = size
 			size = min(size, 5)
```

Comment:
why do we min this exactly?

Reply:
Probably to prevent invisible sprites if there is an ibject with a size larger than 5.

Reply:
Should I add a comment to that to specify why it is capped?

Reply:
Nah, I'm just blind and missed the icon state use